packages/core/src/virtual-entrypoint.ts

```diff
@@ -330,7 +330,7 @@ function shouldSplitRoute(routeName: string, splitAtRoutes: (RegExp | string)[]
 
 export function getAppFiles(appRoot: string): Set<string> {
   const files: string[] = walkSync(appRoot, {
-    ignore: ['_babel_filter_.js', 'app.js', 'assets', 'testem.js', 'node_modules'],
+    directories: false,
```

Comment:
I removed the ignore because we now target the app folder as our appRoot so we don't need these ignores, and I added directories false because we only care about files 😂